lms/static/sass/views/_login-register.scss

```diff
@@ -67,7 +67,7 @@ $sm-btn-linkedin: #0077b5;
         }
     }
 
-    form {
+    form, .hint-wrapper {
```

Comment:
You're really compounding this selector (especially with https://github.com/edx/edx-platform/pull/8603). Please place each new selector on a different line for better legibility with that in mind.